packages/typeorm/src/decorators/kuai-entity.decorator.ts

```diff
@@ -0,0 +1,26 @@
+import { Entity, EntityOptions } from 'typeorm'
+
+export enum Location {
+  Data = '_data',
+  Witness = '_witness',
+}
+
+export const KuaiEntity =
+  (nameSuffix: Location, nameOrOptions?: string | EntityOptions, maybeOptions?: EntityOptions) =>
```

Comment:
just suggestion

It feels like the nameSuffix should be part of the nameOrOptions, and the type of the nameSuffix is Location feels a bit strange

Reply:
`nameOrOptions` and `maybeOptions` are the parameters of `typeorm.Entity`. I will add code comments.